server/views/applications/pages/layout.njk

```diff
@@ -21,7 +21,12 @@
 
 {% block content %}
 
-  {% if page.previous() %}
+  {% if page.previous() === 'dashboard' %}
```

Comment:
Should we default to going back to the dashboard if there's no back link rather than having `dashboard`? I can't think of any circumstance where we won't have a back link.

Reply:
So the exception to the rule is the first page after the CRN lookup (Sentence Type) which you can't go back on. 
Although the code isn't as nice I thought it might be clearer to people other than us where the link is going to have it explicitly say 'dashboard' (although we don't do this for the `next` method. 
I could do something different though EG have an explicit 'none' returned from the `previous` method mean we don't show a backlink, if you think that would be better?